packages/platforms/cloudflare-workers/src/actor_driver.ts

```diff
@@ -7,6 +7,10 @@ export class CloudflareWorkersActorDriver implements ActorDriver {
 		this.#doCtx = ctx;
 	}
 
+	get context(): unknown {
```

Comment:
Afaik this should be `{ state: DurableObjectState, env: unknown }`. There's no other way to access `env` without some nasty function nesting.

Reply:
Required a small update to the impl of the driver (needed to pass in `env`).